### PR TITLE
Create health-education-research

### PR DESCRIPTION
**Pull request by CRMTH**

### CODE REVIEW

File: health-education-research.csl [hunk 1]
```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Health Education Research</title>
+    <id>http://www.zotero.org/styles/health-education-research</id>
+    <link href="http://www.zotero.org/styles/health-education-research" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="" rel="documentation"/>
```
Comment (rmzelle) [inline health-education-research.csl]: @CRMTH, please add the "documentation" link here. Seem like it should be http://www.oxfordjournals.org/our_journals/healed/for_authors/index.html.

File: health-education-research.csl [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    <category citation-format="numeric"/>
```
Comment (rmzelle) [inline health-education-research.csl]: Please add a line here with `<category field="medicine"/>`

Comment (rmzelle) [inline health-education-research.csl]: Also please add lines to add the ISSNs:

```
<issn>0268-1153</issn>
<eissn>1465-3648</eissn>
```

**Comment by CRMTH** [comment]
Added documentation, category field, issns, and deleted summary line. Thanks!

**Comment by rmzelle** [comment]
Perfect, thanks.

**Pull request merged**